### PR TITLE
67: Use XCTContext runActivityNamed:block: to insert the step's title in the report

### PR DESCRIPTION
**Pull request by TitouanVanBelle**
This pull request tries to answer issue #67's needs and adds step titles in the report. It uses Xcode 9's new class XCTContext and therefore doesn't work on Xcode 9. 

Here's how it looks on Xcode 9

screen shot 2017-06-21 at 17 36 47

Here's how it looks on Xcode 8

screen shot 2017-06-21 at 17 39 33

Let me know what you think

### CODE REVIEW

**Comment by brentleyjones** [comment]
I'll get to this after the 4th of July. Busy at work and then a small vacation.

File: Cucumberish/Core/Managers/CCIStepsManager.m [hunk 1]
```diff
@@ -181,7 +181,29 @@ - (void)executeStep:(CCIStep *)step inTestCase:(id)testCase
         implementation.type = @"And";
     }
 
-    implementation.body(implementation.matchedValues, implementation.additionalContent);
+    id xctContextClass = NSClassFromString(@"XCTContext");
+    if (xctContextClass) {
+        SEL aSelector = NSSelectorFromString(@"runActivityNamed:block:");
+
+        id block = ^(id activity) {
```
Comment (brentleyjones) [inline Cucumberish/Core/Managers/CCIStepsManager.m]: You an pull out this block before the check and then execute it in the `else` case. That way if we need to change what it does we only change it in one spot. Also, it makes the `if` case have a single responsibility of wrapping the activity.

Comment (TitouanVanBelle) [inline Cucumberish/Core/Managers/CCIStepsManager.m]: Good idea. Done

**Comment by brentleyjones** [comment]
Thanks @TitouanVanBelle!

**Pull request merged**